### ISSUE TITLE
Migrate (update) database view query

### ISSUE THREAD

**Issue by Keithwachira** [vitusortner/floor#400]
i have an issue with adding a new column to the databaseview and updating the query in a databaseview.Whenever i add a new column or update the databaseview query,the change does not take place until i uninstall the app.This is after even running the build_runner.

**Comment by mqus** [comment]
Yeah, new changes to entities and views are not automatically migrated. Although views could be migrated automatically with no downsides this wasn't implemented yet. You'll either have to write a migration or delete the app data between version updates of your app.

**Comment by Keithwachira** [comment]
Thank you for the response.....anyway i have found a way to solve this.
create a migration that will drop the view and recreate it a fresh.

`1.await database.execute('DROP VIEW IF EXISTS view_name');`
`2.await database.execute('CREATE VIEW IF NOT EXISTS `viewname........`

**Comment by thanhnamitit** [comment]
thank @Keithwachira !

**Comment by SteveCruise** [comment]
mark
